org.open_power.HardwareIsolation.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=open-POWER Host HardwareIsolation
+Wants=mapper-wait@-xyz-openbmc_project-inventory.service
+After=mapper-wait@-xyz-openbmc_project-inventory.service
+
+[Service]
+ExecStart=/usr/bin/env openpower-hw-isolation
+SyslogIdentifier=openpower-hw-isolation
```

Comment:
this may not be needed?

Reply:
This is required right to see the service name in trace?
https://www.freedesktop.org/software/systemd/man/systemd.exec.html